python/plugins/processing/tests/testdata/gdal_algorithm_tests.yaml

```diff
@@ -733,3 +733,23 @@ tests:
       OUTPUT:
         name: expected/gdal/points_along_lines.gml
         type: vector
+
+  - algorithm: gdal:dissolve
```

Comment:
Instead of these tests, we've moved the gdal provider to use command line based tests (which are much easier to debug!). 

See e.g https://github.com/qgis/QGIS/blob/master/python/plugins/processing/tests/GdalAlgorithmsTest.py#L1195

Reply:
Is there a guide/manual about this new kind of test?
I've followed https://github.com/qgis/QGIS/blob/master/python/plugins/processing/tests/README.md and http://qgis.it/processing_tests.html in order to write the tests.

Reply:
Unfortunately not -- it's just a matter of copying the existing ones and adapting. I'm happy to write the initial tests for existing dissolve behavior if you need.